backend/middleware/src/SocketEvents.js

```diff
@@ -56,26 +56,27 @@ function applyEventListeners(io) {
       logger.debug(clients);
     });
 
-    socket.on('timespan_request', (requestedResources) => {
+    socket.on('timespan_request', (requestedResource) => {
       const requestId = uuid();
       const request = {
         requestId,
         clientId,
       };
       let reqRes = null;
```

Comment:
Not very descriptive variable name, should probably be something like `requestedResourceParsed`